### PR TITLE
Add Docker Compose

### PR DESCRIPTION
**Pull request by Blake-Haydon**
## Screenshots

Screen Shot 2021-05-29 at 3 21 43 pm

## Steps to Test

1.

### CODE REVIEW

**Review by hallgchris — COMMENTED** [review]
Thanks for this! Can you add some instructions on how to use to the PR and readme?

**Review by hallgchris — CHANGES_REQUESTED** [review]
Thanks for this! Can you add some instructions on how to use to the PR and readme?

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+version: '3.8'
+services:
+  mosquitto-mqtt-server:
+    image: "eclipse-mosquitto"
+    ports:
+        - "1883:1883"
+    container_name: dashboard-mqtt-server    
+    volumes:
+        - "./.mosquitto:/mosquitto"
```
Comment (kdav108) [inline docker-compose.yml:9]: What does volumes represent? my quick google search wasn't clear enough for me

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+version: '3.8'
+services:
+  mosquitto-mqtt-server:
+    image: "eclipse-mosquitto"
+    ports:
+        - "1883:1883"
```
Comment (kdav108) [inline docker-compose.yml:6]: Is this format (`1883:1883`) a convention or does the two numbers represent different stuff?

**Review by kdav108 — CHANGES_REQUESTED** [review]
From my basic understanding, this looks good, but as Chris mentioned already, it would be good to document this magical stuff.

**Pull request closed without merging**